env0/resource_environment.go

```diff
@@ -86,6 +88,20 @@ func resourceEnvironment() *schema.Resource {
 				Description: "id of the last deployment",
 				Computed:    true,
 			},
+			"ttl": {
+				Type:        schema.TypeString,
+				Description: "the date the environment should be destroyed at (iso format)",
+				Optional:    true,
+				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
+					utcPattern := `\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d`
+					ttl := val.(string)
+					matched, err := regexp.MatchString(utcPattern, ttl)
+					if !matched || err != nil {
+						errs = append(errs, fmt.Errorf("%q must be of iso format, got: %q", key, ttl))
```

Comment:
Maybe add example to ISO date to the error

Reply:
4d01ba2 👍🏻